File: src/lib/ir/__fixtures__/java/08-labeled-break.java
class LabeledBreak {
    int firstZeroRow(int[][] grid) {
        int found = -1;
        outer:
        for (int i = 0; i < grid.length; i++) {
            for (int j = 0; j < grid[i].length; j++) {
                if (grid[i][j] == 0) {
                    found = i;
                    break outer;
                }
            }
        }
        return found;
    }
}

File: src/lib/ir/__fixtures__/js/08-labeled-break.js
function findZero(grid) {
  let found = -1;
  outer:
  for (let i = 0; i < grid.length; i++) {
    for (let j = 0; j < grid[i].length; j++) {
      if (grid[i][j] === 0) {
        found = i;
        break outer;
      }
    }
  }
  return found;
}
